diff --git a/neslysici.me b/neslysici.me
index 423035c..273bf4f 100644
--- a/neslysici.me
+++ b/neslysici.me
@@ -1118,58 +1118,71 @@ a poté jednotlivými \[Bq]poznatky\[lq] z obecné teorie
 vysvětlit empirické jevy spjaté s prostředím Neslyšících.
 .sh 1 "Kdo potřebuje kulturu?"
 .sh 2 "Co je to kultura?"
-.pp
+
 .b "Kultura je:"
+.ls 1
 .(q
 .sz 12
 souhrn duchovních a materiálních hodnot 
 vytvořených a vytvářených lidstvem v celé jeho historii
-.br
+.)q
 .(c
 (ASCS)
 .)c
-.br
+.(q
 soubor hodnot vytvořených tělesnou i duševní činností lidí
 a utvářejících lidskou společnost; civilizace
-.br
+.)q
 .(c
 (SSČ)
 .)c
-.br
+.(q
 soubor výsledků veškeré tělesné a duševní činnosti
 utvářející lidskou společnost
-.br
+.)q
 .(c
 (SSJČ)
 .)c
-.br
-1) celkový způsob života lidí;
-2) společenské dědictví, které jednotlivec získává od své skupiny;
-3) způsob myšlení, cítění a víry;
-4) abstrahování z chování;
-5)
+
+.(q
+.np
+celkový způsob života lidí;
+.np
+společenské dědictví, které jednotlivec získává od své skupiny;
+.np
+způsob myšlení, cítění a víry;
+.np
+abstrahování z chování;
+.np
 .i "antropologova teorie o skutečném způsobu chování skupiny lidí"
-6) zásobárna společných znalostí;
-7) soubor ustálených orientačních reakcí na opakující se problémy;
-8) naučené chování
-9) mechanismus normativní regulace chování
-10) soubor postupů přizpůsobování se vnějšímu prostředí
+.np
+zásobárna společných znalostí;
+.np
+soubor ustálených orientačních reakcí na opakující se problémy;
+.np
+naučené chování
+.np
+mechanismus normativní regulace chování
+.np
+soubor postupů přizpůsobování se vnějšímu prostředí
 a ostatním lidem;
-11) kondenzát historie;
-.br
+.np
+kondenzát historie;
+.)q
 .(c
 (Clyde Kluckhohn, Mirror for Man, 1949)
 .)c
-.br
+.(q
 \[Bq]Domnívaje se,
 [...] že člověk je zvíře zavěšené do pavučiny významů,
 kterou si samo upředlo,
 považuji kulturu za tyto pavučiny [...].\[lq]
-.br
+.)q
 .(c
 (Clifford Geertz, Interpretace kultur, 1973)
 .)c
-.)q
+.ls 2
+
 .pp
 Co mají tyto definice společného?
 Ukazují kulturu jako interpretační rámec podmíněný společností,
